cost/aws/instance_cloudwatch_utilization/README.md

```diff
@@ -1,83 +1,85 @@
+This Policy Template gathers AWS instances with inefficient utilization using CloudWatch CPU and Memory Metrics over a 30 day average and downsized after approval.
 
-### Cloud Management Required Permissions/AWS Required Permissions
-- Cloud Management - The `credential_viewer`,`observer` roles
-- Cloud Management - The `policy_designer`, `policy_manager` & `policy_publisher` roles
-- AWS - The `CloudWatchReadOnlyAccess` AWS IAM Policy
+## Functional Details
 
-### Functional Details
+This policy identifies all running instances reporting performance metrics to CloudWatch whose CPU or Memory utilization is below the thresholds set in the **Average used memory percentage** and **Average used CPU percentage** parameters.
 
-- This policy identifies all instances reporting performance metrics to CloudWatch whose CPU or Memory utilization is below the thresholds set in the **Average used memory percentage** and **Average used CPU percentage** parameters.
-- The **Exclusion Tag Key** parameter is a string value.  Supply the Tag Key only.  Tag Values are not analyzed and therefore are not need.  If the exclusion tag key is used on an Instance, that Instance is presumed to be exempt from this policy.
-- This policy sets the tag defined in the **Action Tag Key:Value** parameter on the underutilized instances that were identified.
-- If you get an **N/A** in a field you will need to install the [CloudWatch Agent](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Install-CloudWatch-Agent.html) on the instance to get those metrics. 
-- This policy only pulls running instances, as it is unable to get correct monitoring metrics from instances in other states.
+The **Exclusion Tag Key** parameter is a string value.  Supply the Tag Key only.  Tag Values are not analyzed and therefore are not need.  If the exclusion tag key is used on an Instance, that Instance is presumed to be exempt from this policy.
 
-#### Windows Support
+Inefficient Instances are resized to the next smaller size within the same class.  If the instance is already on the smallest size for the class it will not be resized. Resize only occurs after approval
 
-To enable windows support you will need to add the following to your cloudwatch config.json and restart cloudwatch agent
-```json
-	"metrics": {
-		"append_dimensions": {
-			"AutoScalingGroupName": "${aws:AutoScalingGroupName}",
-			"ImageId": "${aws:ImageId}",
-			"InstanceId": "${aws:InstanceId}",
-			"InstanceType": "${aws:InstanceType}"
-    }
-  }
-```
-#### Input Parameters
+If you get an **N/A** in a field you will need to install the [CloudWatch Agent](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/Install-CloudWatch-Agent.html) on the instance to get those metrics.
+
+## Policy Actions
+
+- Sends an email notification
+- Downsizes instances after approval
+
+## Prerequisites
+
+This policy requires the AWS IAM User Credential. When applying the policy select the appropriate credentials
```

Comment:
Remove IAM User.  The user can use an AWS IAM or AWS STS Credential to authenticate.

Reply:
Updated as mentioned in the comment